docs/ref/class-based-views/flattened-index.txt

```diff
@@ -168,9 +168,9 @@ FormView
 * :meth:`~django.views.generic.edit.FormMixin.form_invalid`
 * :meth:`~django.views.generic.edit.FormMixin.form_valid`
 * :meth:`~django.views.generic.edit.ProcessFormView.get`
-* :meth:`~django.views.generic.base.ContextMixin.get_context_data`
 * :meth:`~django.views.generic.edit.FormMixin.get_form`
 * :meth:`~django.views.generic.edit.FormMixin.get_form_kwargs`
+* :meth:`~django.views.generic.edit.FormMixin.get_context_data`
```

Comment:
Yeah, that's true; I thought if they were alphabetized it would make sense if the entire list was, at any rate right now they are both in-call-order and alphabetized, so no changes required I think?